### DISCUSSION TITLE
Css Class / Column size per field editor

### DISCUSSION THREAD

**Discussion by MikeKry** [OrchardCMS/OrchardCore#13128] [Q&A]
Hi, 

I would like to ask, if it is possible to apply css classes to fields in CMS?  Because I do not want fields in CMS streched to full width. Instead I would like to set width to e.g. 50% so they would be placed on single line. 

Example:
![image](https://user-images.githubusercontent.com/52829889/213875449-01c2f113-4d28-46bf-a2c5-20fbce032581.png)
after applying col-lg-3
![image](https://user-images.githubusercontent.com/52829889/213875475-32039245-b5a5-4aee-b2e1-c5e814d9c9c5.png)

I tried using placements with no luck and using custom views is way too time consuming.

**Comment by ns8482e** [comment]
every field has css class `field-wrapper` and ` field-wrapper-partname-fieldname`.
you can define your own CSS rule for  `field-wrapper`  or ` field-wrapper-partname-fieldname` or both.
